src/wazuh_db/wdb.c

```diff
@@ -907,6 +909,18 @@ void wdb_close_old() {
     w_mutex_unlock(&pool_mutex);
 }
 
+int wdb_exec_stmt_silent(sqlite3_stmt* stmt) {
+    switch (wdb_step(stmt)) {
+    case SQLITE_ROW:
+    case SQLITE_DONE:
+        return OS_SUCCESS;
+        break;
+    default:
+        mdebug1("SQL statement execution failed");
```

Comment:
We could add a **mdebug2** message with the SQL statement that failed

Reply:
The **stmt** parameter doesn't have a piece of human-readable information to be printed. But I think it's not necessary because it's information that can be obtained from the context of the log.

Reply:
OK !